### PR TITLE
FFFF-PZ bug fixes

### PR DESCRIPTION
**Pull request by profxj**
Fixes 

## Description of the Change

This PR pairs with one in chime_ffff_pz to fix a number
of bugs..  Especially those related to CHIME

## Checklist

Please check all that apply to your proposed changes

- [ ] HTML code change
- [ ] Added package dependency
- [ ] Added data
- [ ] Changed django model
- [ ] Documentation change

### CODE REVIEW

**Comment by profxj** [comment]
Let's not merge this in until we confirm the changes
for `Too Faint` are ok.

File: YSE_App/models/frbtransient_models.py [hunk 1]
```diff
@@ -227,14 +229,37 @@ def sum_top_two_PATH(self):
             path_values = path_values[argsrt]
             return np.sum(path_values[-2:])
 
+    @property
+    def mag_top_two_PATH(self):
+        """ PATH mag for the top two PATH P(O|x) values for the transient 
+
+        If there are more than 1, take the top two and
```
Comment (tarraneh) [inline YSE_App/models/frbtransient_models.py:236]: Do we really want the brightest? In `frb_status.py` it says: 
`            if frb.mag_top_two_PATH > mr_max:
                frb.status = TransientStatus.objects.get(name='TooFaint')
                frb.save()
                return`

But shouldn't this criterion be set by the faintest source? i.e., brightest source may be < mr_max, while faintest is not...

**Review by lordrick94 — APPROVED** [review]
Looks Great :)

File: YSE_App/models/frbtransient_models.py (hunk 1 above)
Comment (profxj) [reply YSE_App/models/frbtransient_models.py:236]: seems right to me.  We are querying whether the sources are `too faint`.  

So, we find the brightest magnitude of the top two PATH and if even that is too faint, we call it too faint.

Am I missing something?

**Review by tarraneh — APPROVED** [review]

**Pull request merged**